CHANGES.md

```diff
@@ -3,6 +3,10 @@
 
 - Add `(deps <deps>)` in ctype field (#5346, @bobot)
 
+- Add `(foreign_archives ((path <path>)(mode whole))` to ensure that all the
```

Comment:
I think that `name` would be better than `path` to go with the existing Dune conventions (since what we write is a "logical" name rather than a filesystem name: no extension, no `lib` or `dll` prefix).